src/test/java/org/broadinstitute/consent/http/service/DarCollectionServiceTest.java

```diff
@@ -1050,38 +1053,58 @@ public void testProcessDarCollectionSummariesForDACMember() {
     DarCollectionSummary summaryTwo = new DarCollectionSummary();
     summaryTwo.addDatasetId(3);
     Election electionThree = new Election();
+    Vote vote = new Vote(1, true, user.getUserId(), null, null, electionThree.getElectionId(), null, VoteType.DAC.getValue(), null, null);
     electionThree.setElectionId(3);
     electionThree.setStatus(ElectionStatus.OPEN.getValue());
     summaryTwo.addElection(electionThree);
+    summaryTwo.addVote(vote);
 
     DarCollectionSummary summaryThree = new DarCollectionSummary();
     summaryThree.addDatasetId(4);
 
+    DarCollectionSummary summaryFour = new DarCollectionSummary();
+    summaryFour.addDatasetId(5);
+    Election electionFour = new Election();
+    Vote voteTwo = new Vote(2, true, user.getUserId(), null, null, electionThree.getElectionId(), null,
+        VoteType.DAC.getValue(), null, null);
+    Vote voteThree = new Vote(4, null, user.getUserId(), null, null, electionThree.getElectionId(), null,
+        VoteType.DAC.getValue(), null, null);
+    electionFour.setElectionId(4);
+    electionFour.setStatus(ElectionStatus.OPEN.getValue());
+    summaryFour.addElection(electionFour);
+    summaryFour.setVotes(List.of(voteTwo, voteThree));
+    
+
     when(darCollectionSummaryDAO.getDarCollectionSummariesForDAC(any(), any()))
-      .thenReturn(List.of(summary, summaryTwo, summaryThree));
+      .thenReturn(List.of(summary, summaryTwo, summaryThree, summaryFour));
     when(datasetDAO.findDatasetDTOsByUserId(any())).thenReturn(Set.of());
 
     initService();
 
     List<DarCollectionSummary> summaries = service.getSummariesForRoleName(user, UserRoles.MEMBER.getRoleName());
 
     assertNotNull(summaries);
-    assertEquals(3, summaries.size());
+    assertEquals(4, summaries.size());
 
     DarCollectionSummary testOne = summaries.get(0);
     Set<String> expectedOneActions = Set.of();
     assertEquals(testOne.getActions(), expectedOneActions);
     assertEquals(DarCollectionStatus.COMPLETE.getValue(), testOne.getStatus());
 
     DarCollectionSummary testTwo = summaries.get(1);
-    Set<String> expectedTwoActions = Set.of("Vote");
+    Set<String> expectedTwoActions = Set.of("Update");
```

Comment:
Minor nit, you might be able to use the existing enums (`DarCollectionActions.UPDATE`) in place of these strings